tasks/base/start-karma.js

```diff
@@ -12,6 +12,6 @@ const task = (cb, options) => {
     }, options || {}), cb).start();
 };
 
-task.displayName = 'karma';
+task.displayName = 'test-client';
```

Comment:
I don't think this change is intended. It will change all calls to karma to read as `test-client` instead. It'll be confusing for errors too since `test-client.js` exists already.